index.js

```diff
@@ -1218,9 +1210,4 @@ exports.main = function() {
     if(typeof simpleStorage.seenUserData === 'undefined') {
         modifyNewtab();
     }
-
-    timers.setTimeout(function() {
```

Comment:
Removed this failsafe for the reset preload preference because it appears that our other check is working fine, and also, this code will never hit on a 24 hour timer unless the user leaves their browser open for 24 hours